### DISCUSSION TITLE
'Document is not defined' when importing bundle on nextJS SSR app

### DISCUSSION THREAD

**Discussion by dariazielinska** [webpack/webpack#16877] [Q&A]
I am importing bundle to my project in next.js and because part of this is server-side rendered, it crashes with error: 'document is not defined' pointing to this part of code:
<img width="617" alt="Screenshot 2023-03-28 at 14 16 16" src="https://user-images.githubusercontent.com/44161041/228233166-8d5d0225-3dda-465f-8802-16661ece53d8.png">
Could someone help to solve this error?

**Comment by alexander-akait** [comment]
You should not use the `web` target for server, because you don't have `document`, sorry, we need more infromation to help you
